libdevice/msvc_wrapper.cpp

```diff
@@ -6,9 +6,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "device.h"
 #include "wrapper.h"
```

Comment:
To be pedantic: `__SPIR_TARGET_ONLY__` is defined in "device.h", so it should be included.

Reply:
`wrapper.h` aggregates everything for libc wrapper implementations.